### PR TITLE
Add conversions for Akka Uri.

### PR DESCRIPTION
**Pull request by wwajerowicz**

### CODE REVIEW

File: modules/akka/src/main/scala/ch/epfl/bluebrain/nexus/rdf/akka/ToAkkaConverters.scala [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    case IriNode(iri) => asAkka(iri)
+    case other        => Left(s"${other.show} cannot be converted to URI.")
+  }
+
+  /**
+    * Attempts to convert argument [[AbsoluteIri]] to Akka [[Uri]].
+    */
+  def asAkka(iri: AbsoluteIri): Either[String, Uri] =
+    Try {
+      Uri.apply(iri.toString)
```
Comment (bogdanromanx) [inline modules/akka/src/main/scala/ch/epfl/bluebrain/nexus/rdf/akka/ToAkkaConverters.scala]: `iri.asUri` would be more appropriate in this case

File: modules/akka/src/main/scala/ch/epfl/bluebrain/nexus/rdf/akka/ToAkkaConverters.scala [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+/**
+  * Conversions from rdf data types and Akka [[akka.http.scaladsl.model.Uri]].
+  */
+trait ToAkkaConverters {
+
+  /**
+    * Attempts to convert argument [[Node]] to Akka [[Uri]].
+    */
+  def asAkka(node: Node): Either[String, Uri] = node match {
```
Comment (bogdanromanx) [inline modules/akka/src/main/scala/ch/epfl/bluebrain/nexus/rdf/akka/ToAkkaConverters.scala]: not sure we need this, but it doesn't hurt ;)

File: modules/akka/src/main/scala/ch/epfl/bluebrain/nexus/rdf/akka/ToAkkaConverters.scala [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    case other        => Left(s"${other.show} cannot be converted to URI.")
+  }
+
+  /**
+    * Attempts to convert argument [[AbsoluteIri]] to Akka [[Uri]].
+    */
+  def asAkka(iri: AbsoluteIri): Either[String, Uri] =
+    Try {
+      Uri.apply(iri.toString)
+    }.toEither.left.map(_ => s"'${iri.toString}' is not a valid URI.")
```
Comment (bogdanromanx) [inline modules/akka/src/main/scala/ch/epfl/bluebrain/nexus/rdf/akka/ToAkkaConverters.scala]: `.left.map` == `.leftMap` if you have `cats.implicits._` imported

File: modules/akka/src/main/scala/ch/epfl/bluebrain/nexus/rdf/akka/ToAkkaConverters.scala [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    */
+  def asAkka(node: Node): Either[String, Uri] = node match {
+    case IriNode(iri) => asAkka(iri)
+    case other        => Left(s"${other.show} cannot be converted to URI.")
+  }
+
+  /**
+    * Attempts to convert argument [[AbsoluteIri]] to Akka [[Uri]].
+    */
+  def asAkka(iri: AbsoluteIri): Either[String, Uri] =
```
Comment (bogdanromanx) [inline modules/akka/src/main/scala/ch/epfl/bluebrain/nexus/rdf/akka/ToAkkaConverters.scala]: not sure we need to wrap this in a `Try`, @umbreak what do you think?

Comment (wwajerowicz) [reply modules/akka/src/main/scala/ch/epfl/bluebrain/nexus/rdf/akka/ToAkkaConverters.scala]: yes, if we use `asUri` we don't need `Try`

**Pull request merged**